policyengine_canada/variables/gov/cra/tax/income/credits/school_supply_credit/school_supply_credit.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_canada.model_api import *
+
+
+class school_supply_credit(Variable):
+    value_type = float
+    entity = Person
```

Comment:
Just confirming this is person and not tax unit level

Is it refundable? Please explain how it'll hook up to the net taxes.

Reply:
Intuit says it was refundable for the prior year, when it was 15% https://turbotax.intuit.ca/tips/teacher-and-early-childhood-educator-school-supply-tax-credit-5573

Reply:
This article implies that the credit is still refundable.
https://www.cbc.ca/news/business/teacher-tax-return-1.6433994